panther_analysis_tool/policy.py

```diff
@@ -0,0 +1,47 @@
+"""
+Panther Analysis Tool is a command line interface for writing,
+testing, and packaging policies/rules.
+Copyright (C) 2020 Panther Labs Inc
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from typing import List
+
+from panther_analysis_tool.enriched_event import PantherEvent
+from panther_analysis_tool.rule import Detection
+
+TYPE_POLICY = "POLICY"
+
+ERROR_TYPE_POLICY = "POLICY_ERROR"
```

Comment:
It seems that nothing is using this variable yet, should it be or is this here against future need?

Reply:
its for the futurrreeee, but a really easy thing to add back in, I'll remove it.